.gitignore

```diff
@@ -43,3 +43,9 @@ tests/venv
 .cache
 
 docs/generated/
+
+/core/cache-main
+/core/cache-tests
+/**/.gitignore
```

Comment:
I retested this after https://github.com/apache/kafka/pull/2382
and even then with the cleaner eclipse structure there are two eclipse generated  gitignores
```
	core/.gitignore
	streams/.gitignore
```